package.json

```diff
@@ -32,7 +36,7 @@
     "lint:scss": "stylelint '{packages,shared}/**/*.scss' --config config/stylelint.config.js",
     "lint:ec": "echint",
     "lint": "npm-run-all --parallel lint:*",
-    "precommit": "lint-staged && yarn test",
+    "precommit": "lint-staged && yarn test && yarn contributors:update",
```

Comment:
Maybe its better for this to be part of the prepush instead? That way you don't regenerate the contributors list every time you make a small commit.

Reply:
Does `yarn contributors:update` only update the README without making a commit?

Reply:
@ryanoglesby08 Sure, sounds good to me.

@theetrain Yeah, exactly.